app/react/docker/containers/CreateView/CreateView.tsx

```diff
@@ -187,9 +187,10 @@ function useOldContainer(initialName?: string) {
       name: [`^/${debouncedName}$`],
     },
   });
+
   useEffect(() => {
-    if (initialName && initialName !== name) {
-      setName(initialName);
+    if (initialName || name) {
+      setName(name !== undefined ? name : initialName);
```

Comment:
but `name` is the current state, why do we need to `setName(name)`?

```ts
if (initialName && !name) {
   setName(initialName)
}
```

I feel this is clearer. The intention of useEffect here was to set it to the initial value when it's loaded.

Reply:
if the container of current `name` exists, then it should be considered as the old container, make sense?

Reply:
yes, but name is set when changing the name field (using the function `syncName`). here you only want to sync the initial container name, no?